.github/workflows/tests.yml

```diff
@@ -9,19 +9,28 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.10', '3.12']
```

Comment:
Similarly, here based on previous comment for maintaining consistency across files.